### DISCUSSION TITLE
Support the new TraceId and SpanId in Serilog

### DISCUSSION THREAD

**Discussion by lindeberg** [serilog-contrib/serilog-sinks-grafana-loki#210] [Ideas]
Serilog 3.1 adds first-class support for `TraceId` and `SpanId`:
https://github.com/serilog/serilog/pull/1955

Each sink has to implement this.

**Comment by mishamyte** [comment]
Thanks for the idea.

Looks like a wonderful feature. But according to the info, it will be the breaking change.
So it should be decided should the new release would be created with a dep from the `Serilog 3.1`. It looks pretty logical to put in into a v9 roadmap, because for me the breaking change should occur only in major releases. And a lot of people, who use that sync, have no possibility to update to the latest `Serilog` in their projects.

So I suggest to put that into a v9 roadmap
